### ISSUE TITLE
'sbt dist ' throws a Missing a swagger.yml or swagger.json to provide base swagger spec

### ISSUE THREAD

**Issue by ferrlin** [iheartradio/play-swagger#190]
When I do sbt run, it works fine but we need to package the play app for deployment.

Either we use `sbt dist` or `sbt universal:package-bin`, it fails with an exception. The log shows me this:

>[error] (run-main-0) com.iheart.playSwagger.SwaggerSpecGenerator$MissingBaseSpecException$: Missing a swagger.yml or swagger.json to provide base swagger spec
com.iheart.playSwagger.SwaggerSpecGenerator$MissingBaseSpecException$: Missing a swagger.yml or swagger.json to provide base swagger spec
	at com.iheart.playSwagger.SwaggerSpecGenerator$MissingBaseSpecException$.<clinit>(SwaggerSpecGenerator.scala)
	at com.iheart.playSwagger.SwaggerSpecGenerator$$anonfun$defaultBase$1.apply(SwaggerSpecGenerator.scala:236)
	at com.iheart.playSwagger.SwaggerSpecGenerator$$anonfun$defaultBase$1.apply(SwaggerSpecGenerator.scala:236)
	at scala.Option.getOrElse(Option.scala:121)
	at com.iheart.playSwagger.SwaggerSpecGenerator.defaultBase$lzycompute(SwaggerSpecGenerator.scala:236)
	at com.iheart.playSwagger.SwaggerSpecGenerator.defaultBase(SwaggerSpecGenerator.scala:236)
	at com.iheart.playSwagger.SwaggerSpecGenerator.generate(SwaggerSpecGenerator.scala:43)
	at com.iheart.playSwagger.SwaggerSpecRunner$.swaggerJson(SwaggerSpecRunner.scala:23)
	at com.iheart.playSwagger.SwaggerSpecRunner$.delayedEndpoint$com$iheart$playSwagger$SwaggerSpecRunner$1(SwaggerSpecRunner.scala:26)
	at com.iheart.playSwagger.SwaggerSpecRunner$delayedInit$body.apply(SwaggerSpecRunner.scala:7)
	at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
	at scala.App$class.main(App.scala:76)
	at com.iheart.playSwagger.SwaggerSpecRunner$.main(SwaggerSpecRunner.scala:7)
	at com.iheart.playSwagger.SwaggerSpecRunner.main(SwaggerSpecRunner.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sbt.Run.invokeMain(Run.scala:67)
	at sbt.Run.run0(Run.scala:61)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:51)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Logger$$anon$4.apply(Logger.scala:84)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)

Would you know how to workaround this? 

Thanks!

**Comment by kailuowang** [comment]
try something like 
```scala
dist := (dist in Compile).dependsOn(swagger).evaluated
```
in your sbt file, see if that works.

**Comment by ferrlin** [comment]
`evaluated` causes an error on sbt so I tried 
`dist := (dist in Compile).dependsOn(swagger).value` but throws an error in sbt when reloading.

I modified it a bit to look something like this:

`dist in Compile := (dist in Compile).dependsOn(swagger).value` and it doesn't throws any error as above but when attempting `sbt dist`, I still get the missing swagger.yml exception. 

In short, it still didn't worked for me.
